plugins/terrain_and_color/main.py

```diff
@@ -27,6 +22,9 @@
 from vistas.ui.utils import *
 
 
+# Todo - resolve color bug
```

Comment:
New bug that I need to address. The shader program doesn't seem to like it when the `noDataValue` is changed, or that it just isn't being updated appropriately. So just some bug hunting to be done here but since it was a rather minor bug overall I left to do later.